packages/tables/frontend/sirius-components-tables/src/table/ClientSideTableContent.tsx

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { Selection, useSelection } from '@eclipse-sirius/sirius-components-core';
+import Box from '@mui/material/Box';
+import { MaterialReactTable, MRT_TableOptions, useMaterialReactTable } from 'material-react-table';
+import { memo, useEffect, useState } from 'react';
+import { SettingsButton } from '../actions/SettingsButton';
+import { GQLLine, ClientSideTableProps } from './TableContent.types';
+import { useTableColumns } from './useTableColumns';
+
+export const ClientSideTableContent = memo(
+  ({ editingContextId, representationId, table, readOnly }: ClientSideTableProps) => {
+    const { selection, setSelection } = useSelection();
+    const [linesState, setLinesState] = useState<GQLLine[]>(table.lines);
+
+    useEffect(() => {
+      setLinesState([...table.lines]);
+    }, [table]);
+
+    const { columns } = useTableColumns(editingContextId, representationId, table, readOnly);
+
+    const tableOptions: MRT_TableOptions<GQLLine> = {
```

Comment:
I've taken a bit more time to understand this but the issue here is not about the pagination server-side vs client-side (or lack of) as I imagined it would be since it was the main issue the last time we discussed about it. Here most features are just entirely removed unless I missed something. What about search, filtering, row resizing, column resizing, row header???

Why is this PR "fixing" something by removing half of its features at the last second before the release?